src/index.ts

```diff
@@ -26,33 +27,61 @@ export default class CAFReceiver {
 
   // Setup DRM if present in `media.customData`
```

Comment:
This looks outdated

Reply:
Still partially true but unnecessary I guess 👍